automation/travisci-test.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash -e
+
+
+make generate
+if [[ -n "$(git status --porcelain)" ]] ; then
```

Comment:
Generally speaking by example,
There is -e, so we wont reach this line (5) if `make generate` fails, and the error will be truncated in a non comfort way, as we saw before, and that was groomed its not the desired behaviour.

Reply:
This PR does one thing as described in the commit message.
This is indeed a problem we need to fix. It should be easy after this change gets in.